posthog/hogql_queries/insights/trends/aggregation_operations.py

```diff
@@ -0,0 +1,147 @@
+from typing import List
+from posthog.hogql import ast
+from posthog.hogql.parser import parse_expr, parse_select
+from posthog.hogql_queries.utils.query_date_range import QueryDateRange
+from posthog.schema import ActionsNode, EventsNode
+
+
+class QueryModifier:
```

Comment:
Perhaps we can rename this to something else? We already have the system of "hogql query modifiers" inside `modifier.py`, which sets things like PoE on/off, cohorts via join/subquery, etc. To avoid confusing people, can we find something else for this? `QueryAlternator` 🙈?

Reply:
Aha, yep! I can change this - shame you can't scope classes to just the file they're defined in